### PR TITLE
fixed NDT plugin

### PR DESCRIPTION
**Pull request by rondomnitz**

### CODE REVIEW

**Comment by rondomnitz** [comment]
bot:retest

**Comment by rondomnitz** [comment]
bot:retest

**Comment by rondomnitz** [comment]
bot:retest

**Comment by rondomnitz** [comment]
bot:retest

**Comment by rondomnitz** [comment]
bot:retest

**Comment by rondomnitz** [comment]
bot:retest

**Comment by rondomnitz** [comment]
bot:retest

**Comment by rondomnitz** [comment]
bot:retest

**Comment by rondomnitz** [comment]
bot:retest

File: plugins/UFM_NDT_Plugin/.ci/ci_matrix.yaml [hunk 1]
```diff
@@ -89,29 +89,58 @@ steps:
...
+      plugins/UFM_NDT_Plugin/.ci/real_fabric.sh
+      opensm_pid=$(pgrep -x opensm)
+      kill -9 $opensm_pid
+      ps -ef | grep -i sm
+    parallel: false
+
+  - name: draft
+    agentSelector: "{nodeLabel: 'UFM-POC'}"
+    run: |
+      plugins/UFM_NDT_Plugin/.ci/draft.sh
```
Comment (atolikin94) [inline plugins/UFM_NDT_Plugin/.ci/ci_matrix.yaml:138]: please use more meaningful name

File: plugins/UFM_NDT_Plugin/.ci/ci_matrix.yaml [hunk 2]
```diff
@@ -89,29 +89,58 @@ steps:
...
   - name: remove plugin end
     agentSelector: "{nodeLabel: 'UFM-POC'}"
-    run: plugins/UFM_NDT_Plugin/.ci/do_remove_plugin.sh || true
+    run: plugins/UFM_NDT_Plugin/.ci/do_remove_plugin.sh
+    parallel: false
+
+  - name: back to real fabric
+    agentSelector: "{nodeLabel: 'UFM-POC'}"
+    run: | 
+      plugins/UFM_NDT_Plugin/.ci/real_fabric.sh
```
Comment (atolikin94) [inline plugins/UFM_NDT_Plugin/.ci/ci_matrix.yaml:129]: as far as I can see, this script just stops UFM, please rename accordingly

File: plugins/UFM_NDT_Plugin/.ci/ci_matrix.yaml [hunk 3]
```diff
@@ -89,29 +89,58 @@ steps:
...
+    run: |
+      plugins/UFM_NDT_Plugin/.ci/draft.sh
+      /etc/init.d/ufmd stop
+      ps -ef | grep -i sm
+      opensm_id=$(cat /var/run/opensm_sim.pid)
+      kill -9 $opensm_id
+      plugins/UFM_NDT_Plugin/.ci/draft.sh
     parallel: false
 
 pipeline_start:
```
Comment (atolikin94) [inline plugins/UFM_NDT_Plugin/.ci/ci_matrix.yaml:146]: please don't remove the build in pipeline_stop, these build artifacts could be useful for debugging

**Review by atolikin94 — CHANGES_REQUESTED** [review]

**Comment by rondomnitz** [comment]
bot:retest

**Comment by rondomnitz** [comment]
bot:retest

**Comment by rondomnitz** [comment]
bot:retest

**Comment by rondomnitz** [comment]
bot:retest

**Comment by rondomnitz** [comment]
bot:retest

**Comment by rondomnitz** [comment]
bot:retest

**Comment by rondomnitz** [comment]
bot:retest

**Comment by rondomnitz** [comment]
bot:retest

**Comment by rondomnitz** [comment]
bot:retest

**Pull request merged**